### PR TITLE
Closes #816: Add maxlength to all input fields.

### LINKED ISSUE #816: If an applicant enters a very long string in a text field, they get a 400
We should guard against this by putting some reasonable max length on strings (10,000 characters?) and returning a helpful validation message if they enter something longer.

### PR DESCRIPTION
**Pull request by ckirksey3**
### Description

I add a maxlength HTML attribute to all LabelTags with a set length of 10,000 characters. This should reduce the likelihood of users submitting responses so large that the form post fails.

The attribute will exist on tag types for which it doesn't make sense (like checkboxes), but this seemed the cleanest way to achieve the protection given the current layout of the code and I don't believe the superfluous attribute will have any effect.

## Release notes:

Enforce max length of 10,000 characters on input fields.

#### General

- [x] Added the correct label: < feature | bug | dependencies | infrastructure | ignore-for-release >
- [x] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

#### User visible changes

- [ ] Wrote browser tests using the [validateAccessibility](https://sourcegraph.com/github.com/civiform/civiform/-/blob/browser-test/src/support/index.ts?L437:14&subtree=true) method
- [x] Manually tested at 200% size
- [ ] Manually evaluated tab order

#### New Features

- [ ] Add new FeatureFlag env vars to `server/conf/application.conf`
- [ ] Conditioned new functionality on a [FeatureFlag](https://docs.civiform.us/contributor-guide/developer-guide/feature-flags)
- [ ] Wrote browser tests with the feature flag off and on, etc.

### Issue(s) this completes

Fixes #816;

### CODE REVIEW

**Comment by nbeloglazov** [comment]
You want to try and add browser test that submits looooong value? Also how does it look visually when you are submitting long value?

**Comment by ckirksey3** [comment]
> Also how does it look visually when you are submitting long value?

If you try to type another character after reaching 10k, it just doesn't type. It's still possible for someone to inject a string larger than 10k into the dom and submit, but this prevents accidental failures.

File: browser-test/src/text.test.ts [hunk 1]
```diff
@@ -114,6 +114,52 @@ describe('Text question for applicant flow', () => {
...
...456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890' +
```
Comment (nbeloglazov) [inline browser-test/src/text.test.ts:146]: I suggest generating 10k string programmattically:

```
let str = '';
for (let i = 0; i < 1000; i++) {
  str += '123456789';
}
```

**Review by nbeloglazov — CHANGES_REQUESTED** [review]

File: browser-test/src/text.test.ts (hunk 1 above)
Comment (ckirksey3) [reply browser-test/src/text.test.ts]: Good call. Updated!

**Review by nbeloglazov — APPROVED** [review]

**Pull request merged**